gateway-protocol/src/main/proto/gateway.proto

```diff
@@ -27,7 +27,34 @@ message HealthResponse {
   repeated BrokerInfo brokers = 1;
 }
 
+message WorkflowRequestObject {
+  string name = 1;
+  enum ResourceType {
+    BPMN = 0;
+    YAML = 1;
+  }
+  ResourceType type = 2;
+  bytes definition = 3;
+}
+
+message DeployWorkflowRequest {
+  repeated WorkflowRequestObject workflows = 1;
+}
+
+message WorkflowResponseObject {
+  string bpmnProcessId = 1;
+  int32 version = 2;
+  int64 workflowKey = 3;
+  string resourceName = 4;
+}
+
+message DeployWorkflowResponse {
+  repeated WorkflowResponseObject workflows = 1;
```

Comment:
No resources? :cry:

Reply:
Not sure what you mean. I didn't wanted to name things exactly the same as the object we already have cause it's a bit ugly to work with them then. I did that initially with health check stuff and personally, I find it way nicer to make a clear separation of concerns.

Reply:
DeploymentEvent (the response object) which the client will receive will have both of them inside and you can access them if needed. That does not mean that we have to marshal them on the wire twice. 

Or I'm missing the point here? 🙈

Reply:
Maybe I'm not understand correctly how grpc works. But for me it seems that in the response object I have no access to the resources. My question is why? 

If the response should not include the resources anymore, then we have no need to send them back from the broker right? 

Take a look into `ResponseManager#toDeployWorkflowResponse` here the response is build to send to the client correct? Only the workflows are used, so no need of the resources?